code/src/layouts/app-header/components/header/Header.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
+import { IntlProvider } from "react-intl";
+import Header from "@/layouts/app-header/components/header/Header";
+
+jest.mock("@/assets/images/logo.jpeg", () => {});

```

Comment:
I think we should add global mock for all images. It can be done via `jest.config.js` like one of the values in `moduleNameMapper` field of config object:

```js
"\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2)$":
      "identity-obj-proxy",
```

We can also mock our styles like following:

```js
"\\.(css|scss)$": "identity-obj-proxy",
```

`identity-obj-proxy` is a popular external package, which is relatively small. I think we can use it. What do you think?

Reply:
We should to discuss it